job-dsls/jobs/kogito.groovy

```diff
@@ -59,7 +60,7 @@ pipeline {
    }
 }'''
 
-pipelineJob("$folderPath/kogito-pipeline-${mainBranch}") {
+pipelineJob("$folderPath/kogito-deploy-pipeline-${mainBranch}") {
```

Comment:
this brought my doubt, see my comment below about deploying daily snapshots to JBoss org repository